src/signUp.js

```diff
@@ -0,0 +1,47 @@
+import { TEST_AUTH, ERROR_MESSAGES } from "./constants/VALIDATION.js";
+import {
+  emailError,
+  emailInput,
+  pwError,
+  pwInput,
+  pwConfirmError,
+  pwConfirmInput,
+  form,
+  pwToggle,
+  pwConfirmToggle,
+  validateEmail,
+  validatePw,
+  validatePwConfirm,
+  eyeToggle,
+  showError,
+} from "./auth.js";
+
+const isDuplicateEmail = () => {
+  if (emailInput.value.trim() === TEST_AUTH.email) {
+    showError(emailError, emailInput, ERROR_MESSAGES.email_duplicate);
+    return true;
+  } else return false;
+};
+
+const handleSignUp = (e) => {
+  e.preventDefault();
+  if (
+    !isDuplicateEmail() &&
+    validateEmail() &&
+    validatePw() &&
+    validatePwConfirm()
+  ) {
+    alert("회원가입 성공!");
+    window.location.href = "folder.html";
+  }
+};
+
+emailInput.addEventListener("focusout", validateEmail);
+emailInput.addEventListener("focusout", isDuplicateEmail);
```

Comment:
event handler 함수를 하나로 통일하는 방법도 있습니다.
```suggestion
const handleEmailInput = () => {
  validateEmail();
  validateEmailDuplication();
}

emailInput.addEventListener("focusout", handleEmailInput);
```